notebooks/Binary Classification with the UCI Credit-card Default Dataset.ipynb

```diff
@@ -227,7 +227,7 @@
     "lgb_params = {\n",
     "    'objective' : 'binary',\n",
     "    'metric' : 'auc',\n",
-    "    'learning_rate': 0.03,\n",
+    "    'eta0': 0.03,\n",
```

Comment:
this is a bug. revert back to `learning_rate`.

Reply:
Yes, found that last week :-) Already changed locally but I ran into some more issues that I'm fixing now.